### DISCUSSION TITLE
igl::triangulate returns edge markers, but not the list of edges

### DISCUSSION THREAD

**Discussion by evouga** [libigl/libigl#1740] [General]
Libigl's triangle wrapper API returns the new edge markers (`out.segmentmarkerlist`) in the variable `EM2`, but does not return the actual list of new segments (`out.segmentlist`). Unless I'm missing something, that defeats the point of `EM2`?

**Comment by jdumas** [comment]
Hmm I can't say that this part of the triangle wrapper has been widely used, so it's possible it's just an omission. @qnzhou recently created a light wrapper around triangle's API that is more feature complete, so you might want to look into this instead:
https://github.com/qnzhou/trianglelite/
